src/main/play-doc/developer/ConductrSandboxCluster.md

```diff
@@ -0,0 +1,70 @@
+# ConductR sandbox cluster
+
+In order to manage a ConductR cluster we provide a sbt plugin [sbt-conductr-sandbox](https://github.com/typesafehub/sbt-conductr-sandbox). Its purpose is to simplify the setup of a ConductR cluster during development or on a continous integration platform.  By default it uses the conductr-dev Docker image which gets automatically pulled down from Typesafe's public docker repository on bintray when starting the cluster.
```

Comment:
In particular, the initial use-case is that a developer needs to be able to test their bundle's configuration prior to deploying into a production environment.